content/08.annex.md

```diff
@@ -1,13 +1,94 @@
-## Annex
+# A1. Annex {.page_break_before}
 
-The Quality Criteria described in this document is abstract, as such the choice of tools and services to implement
-their verification is up to the team or community developing and/or using a given software.
+The Quality Criteria described in this document is abstract, as such the choice of tools and
```

Comment:
Use "technology-agnostic" instead of "abstract", such as: 
> "The Quality Criteria described in this document follows a technology-agnostic approach, as such .."